common/source/docs/common-efi.rst

```diff
@@ -87,3 +87,12 @@ First, set either RPM1 or RPM2 sensor as being sourced from the EFI (examples wi
 
 - set :ref:`RPM1_TYPE<RPM1_TYPE>` = 3 (EFI)
 - then setup the :ref:`common-rpm-based-notch`
+
+EFI connected via MAVLink
+==============================
+
+Some EFIs do not use the autopilot as their computer, those EFIs have their own onboard computer. It might be possible to connect those EFIs to the autopilot by a MAVLink connection (e.g. then it is possible to read the EFI's status, temperature, etc. by a LUA script). The following parameter should be set to connect an EFI via MAVLink:
```

Comment:
```suggestion
Some EFIs connect to the autopilot via MAVLink, instead of by a Serial port or DroneCAN.  The following parameter should be set to connect such an EFI via MAVLink:
```